modules/custom/d_p_side_tiles/templates/paragraph--d-p-side-tiles.html.twig

```diff
@@ -69,14 +69,14 @@
     <div {{ attributes.addClass(classes) }}>
       <div {{ tiles_wrapper_attributes.addClass(tiles_wrapper_classes) }}>
         <div {{ images_side_attributes.addClass(images_side_classes) }}>
-          {{ content.group_d_side_tiles_gallery }}
+          {{ content.group_d_tiles_content.field_d_media_image }}
         </div>
         <div {{ content_side_attributes.addClass(content_side_classes) }}>
           <div {{ content_fields_attributes.addClass(content_fields_classes) }}>
-            {{ content.group_d_side_tiles_content }}
-            {% if content.group_d_side_tiles_cta %}
+            {{ content.group_d_tiles_content|without('field_d_media_image', 'field_d_cta_link') }}
+            {% if content.group_d_tiles_cta %}
```

Comment:
Are we sure we want to name it as `group_d_tiles`? IMO it can be misleading, as it is `d_p_side_tiles` module

Reply:
@grzegorz-pietrzak-droptica that is why I wanted to have a call with you for this PR review :)